src/format/parsed.rs

```diff
@@ -349,36 +352,50 @@ impl Parsed {
         // it is consistent with other given fields.
         let (verified, parsed_date) = match (given_year, given_isoyear, self) {
             (Some(year), _, &Parsed { month: Some(month), day: Some(day), .. }) => {
-                // year, month, day
                 let date = NaiveDate::from_ymd_opt(year, month, day).ok_or(OUT_OF_RANGE)?;
                 (verify_isoweekdate(date) && verify_ordinal(date), date)
             }
-
             (Some(year), _, &Parsed { ordinal: Some(ordinal), .. }) => {
-                // year, day of the year
                 let date = NaiveDate::from_yo_opt(year, ordinal).ok_or(OUT_OF_RANGE)?;
                 (verify_ymd(date) && verify_isoweekdate(date) && verify_ordinal(date), date)
             }
-
             (Some(year), _, &Parsed { week_from_sun: Some(week), weekday: Some(weekday), .. }) => {
-                // year, week (starting at 1st Sunday), day of the week
                 let date = resolve_week_number(year, week, Weekday::Sun, weekday)?;
                 (verify_ymd(date) && verify_isoweekdate(date) && verify_ordinal(date), date)
             }
-
             (Some(year), _, &Parsed { week_from_mon: Some(week), weekday: Some(weekday), .. }) => {
-                // year, week (starting at 1st Monday), day of the week
                 let date = resolve_week_number(year, week, Weekday::Mon, weekday)?;
                 (verify_ymd(date) && verify_isoweekdate(date) && verify_ordinal(date), date)
             }
-
             (_, Some(isoyear), &Parsed { isoweek: Some(isoweek), weekday: Some(weekday), .. }) => {
-                // ISO year, week, day of the week
-                let date = NaiveDate::from_isoywd_opt(isoyear, isoweek, weekday);
-                let date = date.ok_or(OUT_OF_RANGE)?;
+                let date =
+                    NaiveDate::from_isoywd_opt(isoyear, isoweek, weekday).ok_or(OUT_OF_RANGE)?;
+                (verify_ymd(date) && verify_ordinal(date), date)
+            }
+            #[rustfmt::skip]
```

Comment:
Why skip formatting there?

Reply:
FWIW I've learned to abide with rustfmt's choices.

Reply:
Maybe I have to learn that yet :laugh:. In some cases rustfmt makes the code less readable, and I try to rewrite it in a way that formats nice again.
Using an extra pair of braces here seems to give pretty nice formatting:
```rust
            (
                (true, Some(year), None),
                &Parsed { month: Some(month), week_from_sun: None, week_from_mon: None, .. },
            ) => {
                let date = NaiveDate::from_ymd_opt(year, month, 1).ok_or(OUT_OF_RANGE)?;
                (verify_isoweekdate(date) && verify_ordinal(date), date)
            }
            (
                (true, Some(year), None),
                &Parsed { week_from_sun: Some(week), week_from_mon: None, month: None, .. },
            ) => {
                let date = resolve_week_number(year, week, Weekday::Sun, Weekday::Sun)?;
                (verify_ymd(date) && verify_isoweekdate(date) && verify_ordinal(date), date)
            }
```